Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7-slim
+FROM python:3.6
```

Comment:
Would it be possible to switch this to `3.6-slim-jessie` ?

Reply:
sure thing